airbyte-integrations/connectors/source-amplitude/setup.py

```diff
@@ -6,10 +6,14 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.1",
```

Comment:
why remove this constraint?

Reply:
GA connectors should always be on latest version of airbyte-cdk

Reply:
practically speaking `~=` allows you to get that effect but still ensures you don't accidentally upgrade to a new major version (which we would expect a breaking change)

Reply:
Yep, but I thought we have always keep our GA up-to-date on latest version CDK, am I right?
Everyday connectors build report helps us in this case.